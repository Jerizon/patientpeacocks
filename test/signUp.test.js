import React from 'react';
import ReactDOM from 'react-dom';
import Signup from '../react-client/src/components/signUp.jsx';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

describe('Component: App', () => {
  const items = ['Learn react', 'rest', 'go out'];

  it('should match its empty snapshot', () => {
    const tree = renderer.create(
      <Signup/>
     ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});