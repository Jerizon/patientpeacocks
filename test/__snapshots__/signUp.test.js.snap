// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should match its empty snapshot 1`] = `
<div
  className="signupPage"
>
  <form
    method="POST"
    onSubmit={[Function]}
  >
    <div
      className="login-text"
    >
      Sign Up
    </div>
    <div
      className="loginpg-text"
    >
      Username: 
      <br />
      <input
        type="text"
      />
    </div>
    <div
      className="loginpg-text"
    >
      Password: 
      <br />
      <input
        type="text"
      />
    </div>
    <div
      className="login-button"
    >
      <input
        className="log-button"
        type="submit"
        value="Signup!"
      />
    </div>
  </form>
  <div
    className="links"
  >
    <a
      onClick={[Function]}
      style={Object {}}
    >
      Click here if you have an account!
    </a>
    <br />
    <a
      onClick={[Function]}
      style={Object {}}
    >
      Back to homepage
    </a>
  </div>
</div>
`;
